### PR TITLE
OTT-3927: MMS Vcard V0 specs

### PR DESCRIPTION
**Pull request by usama1020**
# Description

# Checklist

- [x] version number incremented (in the `info` section of the spec)

### CODE REVIEW

File: definitions/messages-olympus.yml [hunk 1]
```diff
@@ -346,6 +355,70 @@ components:
...
+                type:
+                  type: string
+                  description: The type of message to be sent
+                  example: image
+                  enum:
+                    - image
+                image:
+                  type: object
+                  required:
+                    - url
```
Comment (AFavreNexmo) [inline definitions/messages-olympus.yml:383]: It could be caption as optional as well, isn't it?

File: definitions/messages-olympus.yml [hunk 2]
```diff
@@ -346,6 +355,70 @@ components:
...
+                  example: vcard
+                  enum:
+                    - vcard
+                vcard:
+                  type: object
+                  required:
+                    - url
+                  properties:
+                    url:
+                      description: The URL of the vcard attachment. The vcard file is available for 48 hours after it is created.
```
Comment (AFavreNexmo) [inline definitions/messages-olympus.yml:418]: Where does this 48 hours come from?

**Review by AFavreNexmo — APPROVED** [review]
Looks good, would be good just to generate the Heroku to see how it looks, but I forgot how to do it.

File: definitions/messages-olympus.yml (hunk 1 above)
Comment (usama1020) [reply definitions/messages-olympus.yml:383]: yh, it is missing. I'll add it.

File: definitions/messages-olympus.yml (hunk 2 above)
Comment (usama1020) [reply definitions/messages-olympus.yml]: i'm not sure. We have it for mms-image message so i guessed it'll be same for vcard.

**Review by AFavreNexmo — APPROVED** [review]
Looks good to me

**Review by AFavreNexmo — APPROVED** [review]
Looks good to me 🚀

**Review by pardel — APPROVED** [review]

**Pull request merged**